include/zephyr/usb_c/usbc.h

```diff
@@ -220,7 +240,105 @@ typedef uint32_t (*policy_cb_get_rdo_t)(const struct device *dev);
  */
 typedef bool (*policy_cb_is_snk_at_default_t)(const struct device *dev);
 /** @}*/
-/** @endcond */
+/** \addtogroup source_callbacks
+ *  @{
+ */
+/**
+ * @brief Callback type used to get the Rp value that should be placed on
+ *        the CC lines
+ *
+ * @param dev USB-C Connector Instance
+ * @param rp
```

Comment:
Missing description for the `rp` parameter.  I thought the CI had checks for this.

Reply:
done